packages/mdc-theme/_variables.scss

```diff
@@ -138,5 +136,8 @@ $mdc-theme-property-values: (
   text-secondary-on-dark: map-get(map-get($mdc-theme-text-colors, light), secondary),
   text-hint-on-dark: map-get(map-get($mdc-theme-text-colors, light), hint),
   text-disabled-on-dark: map-get(map-get($mdc-theme-text-colors, light), disabled),
-  text-icon-on-dark: map-get(map-get($mdc-theme-text-colors, light), icon)
+  text-icon-on-dark: map-get(map-get($mdc-theme-text-colors, light), icon),
+  text-textfield-background-on-dark: map-get(map-get($mdc-theme-text-colors, light), textfield-background),
```

Comment:
Should these be in the new mdc-textfield/_variables.scss instead, especially now that mdc-theme-prop can accept colors? I think it should be feasible to leave mdc-theme completely untouched in this PR.

Reply:
+1

Reply:
You might be right.